UniversalDashboard.Forge.psm1

```diff
@@ -117,36 +117,44 @@ function New-UDDesktopApp {
         if ($PathInfo.PSIsContainer)
         {
             Write-Verbose "Copying contents of $Path to $src"
-            Copy-Item -Path "$($PathInfo.FullName)/*" -Destination $src -Container -Recurse
+            Copy-Item -Path "$($PathInfo.FullName)/*" -Destination $src -Exclude (Split-Path $OutputPath -Leaf) -Recurse
         }
 
         Write-Verbose "Copying dashboard and index.js to electron src folder: $src"
 
-        $Content = Get-Content $Dashboard -Raw
-        $Content = "
-        `$Env:PSModulePath = `$Env:PSModulePath + `";`$PSScriptRoot`"
-        Import-Module UniversalDashboard" + [System.Environment]::NewLine + $Content
-        $Content | Out-File (Join-Path $Src "dashboard.ps1") -Force -Encoding utf8
+        $content = Get-Content $dashboard -Raw
+        $content = @"
+{0} = {0} + "; {1}"
+Import-Module UniversalDashboard
+
+{2}
+"@ -f '$Env:PSModulePath', '$PSScriptRoot', $content
```

Comment:
I added the here-string with the -f operator because I feel like it's generally easier to read. I think escaping characters with a back tick makes things harder to parse and understand what's happening in the code usually.